resources/views/dashboard.blade.php

```diff
@@ -156,34 +156,34 @@
 
                             <div class="progress">
                                 <div class="progress-bar progress-bar-yellow" role="progressbar" aria-valuenow="60" aria-valuemin="0" aria-valuemax="100" style="width: 60%">
-                                    <span class="sr-only">60% Complete (warning)</span>
```

Comment:
I can't actually get this particular line to display. I am guessing it only shows in some sort of compatibility mode when the html doesn't render correctly. But, I turned it into a phrase anyway.

Reply:
`sr-only` means `screen reader only` - it's not meant to show up on the screen.